src/navigate-pop/using-workspaces.md

```diff
@@ -0,0 +1,48 @@
+# Workspaces
+
+Workspaces serve to reduce window clutter, especially on smaller displays. 
+
+---
+
+## Using Workspaces
+
+Show and access workspaces by clicking on the workspaces icon in the dock, clicking `Workspaces` in the top left corner of your screen, or by pressing `SUPER` + `D`.
+
+The open windows for a given workspace will display when that workspace is selected.
+
+<video autoplay loop>
+        <source src="/images/workspaces/show-workspaces.webm" />
+</video>
+
+Click and drag applications between workspaces as needed.
+
+<video autoplay loop>
+        <source src="/images/workspaces/drag-window.webm" />
```

Comment:
Again, the window being minimized means it's not showing up in the preview, which makes the previews look pointless and not provide any visual feedback.